### ISSUE TITLE
make internal bleeding quite a bit more intense

### ISSUE THREAD

**Issue by fhgb54643689** [studio-minus/ppg-bugs-and-requests#860]
**Feature description**
as the title says please make internal bleeding a lot more intense because it almost feels like its not there.

**Feature purpose**
right now internal bleeding basically does nothing but make limbs a bit more numb so it would be cooler if it had a higher chance of lethality aka lowering the integrity faster

**Additional context**
none.

**Comment by nonbinarynerd** [comment]
Agreed, I'd suggest it also lowering blood pressure, since while the blood may still be in the body it's still leaving the human's veins. Maybe also in certain areas it being an instant death(i.e. the head, since it's got a lot of veins, and if the brain doesn't receive blood it tends to not be very good)
